package/rpm/SOURCES/usr/sth/conf/sth_default.conf

```diff
@@ -0,0 +1,71 @@
+# for further information
+
+# Note: is important to export environment variables with export
+
+# The host where the STH server will be started. Default value: "localhost".
+export STH_HOST="0.0.0.0"
+
+# The port where the STH server will be listening. Default value: 8666.
+export STH_PORT="8666"
+
+# The logging level of the messages. Messages with a level equal or superior 
+# to this will be logged. Allowed values are: .Default value: "info".
+export LOG_LEVEL="info"
+
+# A flag indicating if the logs should be sent to the console. Default value: true.
+export LOG_TO_CONSOLE="false"
+
+# A flag indicating if the logs should be sent to a file. Default value: true.
+export LOG_TO_FILE="true"
+
+# Maximum size in bytes of the log files. If the maximum size is reached, a new log file is created incrementing a counter used as the suffix for the log file name. Default value: undefined.
+# 20MB=20*1024*1024=20971520
+export LOG_FILE_MAX_SIZE_IN_BYTES="20971520"
+
+# The path to a directory where the log file will be searched for or created if it does not exist. Default value: "./log"
+export LOG_DIR="/var/log/sth"
+
+# The name of the file where the logs will be stored. Default value: "sth_app.log".
+export LOG_FILE_NAME="sth_default.log"
+
+# The prefix to be added to the service for the creation of the databases. More information below. Default value: "sth".
+export DB_PREFIX="sth"
+
+# The service to be used if not sent by the Orion Context Broker in the notifications. Default value: "orion".
+export SERVICE="orion"
+
+# The prefix to be added to the collections in the databases. More information below. Default value: "sth".
+export COLLECTION_PREFIX="sth"
+
+# The service path to be used if not sent by the Orion Context Broker in the notifications. Default value: "/".
+export SERVICE_PATH="/"
+
+# The default MongoDB pool size of database connections. Default value: "5".
+export POOL_SIZE="5"
+
+# The data model to use. Currently 3 possible values are supported: collection-per-service-path (which creates a MongoDB collection per service patch to store the data), collection-per-entity (which creates a MongoDB collection per service path and entity to store the data) and collection-per-attribute (which creates a collection per service path, entity and attribute to store the data). More information about these values below. Default value: "collection-per-attribute".
+export DATA_MODEL="collection-per-attribute"
+
+# The username to use for the database connection. Default value: "".
+export DB_USERNAME=""
+
+# The password to use for the database connection. Default value: "".
+export DB_PASSWORD=""
+
+# The URI to use for the database connection. This does not include the 'mongo://' protocol part. Two or more host are comma separated. Default value: "localhost:27017".
+export DB_URI="localhost:27017"
+
+# The name of the database to use. Default value: "test".
+export DB_NAME="sth"
```

Comment:
OK, I review the sth_configuration.js file and I find the next changes:
- `SERVICE` is replaced by `DEFAULT_SERVICE`.
-  `SERVICE_PATH` is replaced by `DEFAULT_SERVICE_PATH`.
- `DB_NAME` has disappeared.
- `REPLICA_SET` is new. Te description of this parameter will be: _The name of the MongoDB replica set. Default value: ""._
- `SHOULD_STORE` and `WRITE_CONCERN` are new variables but I don't know the meaning :-(

I will also update main `README.md` with the new configuration variables when I have the description of the `SHOULD_STORE` and `WRITE_CONCERN`